README.md

```diff
@@ -341,6 +344,7 @@ Currently, the following report formats are supported (reporter names are case-i
 * [Antenna Attribution Document (PDF)](./docs/reporters/AntennaAttributionDocumentReporter.md) (`-f AntennaAttributionDocument`)
 * [CycloneDX](https://cyclonedx.org/) BOM (`-f CycloneDx`)
 * [Excel](https://products.office.com/excel) sheet (`-f Excel`)
+* [GitLabLicenseModel](https://docs.gitlab.com/ee/user/compliance/license_compliance/#configuration) (`-f GitLabLicenseModel`)
```

Comment:
Maybe it better to link to https://docs.gitlab.com/ee/ci/pipelines/job_artifacts.html#artifactsreportslicense_scanning-ultimate as artifact ORT produces is a `artifacts:reports:license_scanning`

Reply:
Done.